Chapter2_ProgramStructure/Chapter 2 #1.JS

```diff
@@ -0,0 +1,14 @@
+function f1()
+{	
+  	var t="";
+    for(var i=1;i<8;i++)
+    {
+        for(var j=1;j<=i;j++)
```

Comment:
For loops are expensive. 

Can you think of a way of doing this in one loop? 

EJS has a hint:
You can start with a program that simply prints out the numbers 1 to 7, which you can derive by making a few modifications to the even number printing example given earlier in the chapter, where the for loop was introduced.

Now consider the equivalence between numbers and strings of hash characters. You can go from 1 to 2 by adding 1 (+= 1). You can go from "#" to "##" by adding a character (+= "#"). Thus, your solution can closely follow the number-printing program.